packages/devtools_app/lib/src/service/service_extensions.dart

```diff
@@ -57,7 +57,6 @@ class ServiceExtensionDescription<T> {
   ServiceExtensionDescription({
     this.iconAsset,
     this.iconData,
-    List<String>? displayValues,
```

Comment:
can you revert changes to this file? we have an outstanding issue to add the UI control for the `togglePlatformMode` service extension, at which point this would need to be added back. https://github.com/flutter/devtools/issues/2780

Reply:
Done